buildSrc/build.gradle

```diff
@@ -7,6 +7,8 @@ repositories {
 }
 
 dependencies {
+    implementation "gradle.plugin.com.github.johnrengelman:shadow:7.1.2"
```

Comment:
should we have a `buildSrc/gradle.properties` to have these versions numbers defined there?

Reply:
I don't think it's worth it for now. Actually we should get rid of `buildSrc` in favor of `build-logic` soon. In that process we could reuse the version catalog too.